doc/man/man1/kdb-find.1

```diff
@@ -71,7 +71,8 @@ kdb find \'fizz\'
 #> user/tests/find/tests/fizz/buzz
 #> user/tests/find/tostfizz
 
-sudo kdb umount /tests/find
+kdb rm \-r /tests/find
+kdb umount /tests/find
```

Comment:
Why is umount here but not above in the source? This seems to be correct.

Reply:
> Why is umount here but not above in the source?

I guess my script to remove those pesky man pages where only the date changed also reverted the (single-line) update to the man page.

The man page and the ReadMe for `kdb-find` should be in sync after the last update again.